imqsauth/http.go

```diff
@@ -770,3 +858,12 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 		httpSendGroupsJson(w, groups)
 	}
 }
+
+func getUserId(r *httpRequest) (authaus.UserId, error) {
```

Comment:
I think there are probably some other places in this source file that could benefit from using this function. I mean.. if it reduces the total lines of code, then it's a win.